docker/nginx/conf.d/default.conf

```diff
@@ -1,6 +1,6 @@
 server {
     server_name localhost;
-    root /var/www/web; ## <-- Your only path reference.
+    root /var/www/html/web; ## <-- Your only path reference.
```

Comment:
I moved the docroot in by one directory level to deal with two issues I was having:

1. For some reason in Windows, Docker was refusing to mount the `/var/www` directory. I believe it was because the `/var/www` directory is not empty inside of Docker, and even though Docker for Linux doesn't care, this was a problem on Windows.
2. `/var/www` is considered the home directory for user `www`. As a result, any temporary files that are generated such as `.bash_history` and `.tmp` were showing up in my local system's project directory. Moving the mounted directory in one level to `/var/www/html` fixed this problem for me.